core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp

```diff
@@ -117,9 +129,12 @@
           "ranger.usersync.ldap.url": "{{ LDAP_URL }}",
           "ranger.usersync.ldap.binddn": "{{ LDAP_BIND_DN }}",
           "ranger.usersync.ldap.ldapbindpassword": "{{ LDAP_BIND_PASSWORD }}",
-          "ranger.usersync.ldap.user.nameattribute": "{{ LDAP_NAME_ATTRIBUTE }}",
-          "ranger.usersync.ldap.user.searchbase": "{{ LDAP_SYNC_SEARCH_BASE }}",
-          "ranger.usersync.group.searchenabled": "false"
+          "ranger.usersync.ldap.user.nameattribute": "{{ LDAP_USERNAME_ATTRIBUTE }}",
+          "ranger.usersync.ldap.user.searchbase": "{{ LDAP_USER_SEARCH_BASE }}",
+          "ranger.usersync.ldap.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
```

Comment:
We don't need search filter, we can use: "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"